src/Resources/views/Action/pay.html.twig

```diff
@@ -0,0 +1,137 @@
+{% extends layout ?: "@PayumCore/layout.html.twig" %}
+
+{% block payum_body %}
+    {{ parent() }}
+
+    <form action="{{ actionUrl|default('') }}" method="POST" name="payment-form" id="payment-form" style="max-width: 500px;" data-secret="{{ model.client_secret }}">
+        <div class="form-row" style="margin: 50px 0;">
+            <div id="card-element">
+                <!-- Elements will create input elements here -->
+            </div>
+
+            <!-- We'll put the error messages in this element -->
+            <div id="card-errors" role="alert"></div>
+        </div>
+
+        <button type="submit">Pay</button>
+    </form>
+{% endblock %}
+
+{% block payum_javascripts %}
+    {{ parent() }}
+
+    <script src="https://js.stripe.com/v3/"></script>
+
+    <script type="text/javascript">
```

Comment:
It would be nice to extract this as a separate js file

Reply:
Agree, but not easily overridable at the end. Maybe dividing this js code into twig blocks will allow more customizations (block names: stripe_js_styles, stripe_js_card, stripe_js_form, stripe_js_error).
I don't see any core payum gateways using it own internal JS and CSS file, maybe it's because payum don't use `symfony/asset`.

Reply:
Then lets go for just twig blocks

Reply:
Agreed!